lib/buildkite/test_collector/http_client.rb

```diff
@@ -41,6 +41,20 @@ def post_json(data)
       http.request(contact)
     end
 
+    def summary_links
```

Comment:
I'm not sure if summary / summary_links is the correct name here. I was imaging in the future we could send through some extra links or into in this to output in the log